src/App.tsx

```diff
@@ -15,36 +15,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alphabetically = 'alphabetically',
+  length = 'length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  { sortField, isReversed }: { sortField: SortType; isReversed: boolean },
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    switch (sortField) {
+      case 'alphabetically':
+        preparedGoods.sort((good1, good2) => good1.localeCompare(good2));
+        break;
+      case 'length':
+        preparedGoods.sort((good1, good2) => good1.length - good2.length);
+        break;
+      default:
+        break;
+    }
+  }
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
The `sortField` state is initialized as an empty string, which is not a valid `SortType`. Consider initializing it with a valid default value from `SortType` to avoid potential runtime errors.